dev-lang/gnucobol/gnucobol-2.2.recipe

```diff
@@ -0,0 +1,123 @@
+SUMMARY="A free COBOL compiler"
+DESCRIPTION="GnuCOBOL (formerly OpenCOBOL) is a free COBOL compiler."
+HOMEPAGE="https://www.gnu.org/software/gnucobol/"
+COPYRIGHT="2001-2017 Free Software Foundation, Inc
+	Keisuke Nishida
+	Roger While
+	Ron Norman
+	Simon Sobisch
+	Edward Hart
+	Brian Tiffin
+	Sergey Kashyrin
+	Dave Pitts"
+LICENSE="GNU GPL v3
+	GNU LGPL v3
+	GNU FDL v1.3"
+REVISION="1"
+SOURCE_URI="https://sourceforge.net/projects/open-cobol/files/gnu-cobol/$portVersion/gnucobol-$portVersion.tar.gz"
+CHECKSUM_SHA256="925838decd65864b2aa3a4bf1385ce4bc708b942e05e8406945a730d7aab32cb"
+
+ARCHITECTURES="!x86_gcc2 ?x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+libVersion="4.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	gnucobol$secondaryArchSuffix = $portVersion
+	cmd:cobc$commandSuffix = $portVersion
+	cmd:cobcrun$commandSuffix = $portVersion
+	lib:libcob$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libdb$secondaryArchSuffix
+	lib:libgmp$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libltdl$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	gnucobol${secondaryArchSuffix}_devel = $portVersion
+	cmd:cob_config$commandSuffix
+	devel:libcob$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	gnucobol$secondaryArchSuffix == $portVersion base
+	devel:libdb$secondaryArchSuffix
+	devel:libgmp$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:libltdl$secondaryArchSuffix
+	devel:libncurses$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libdb$secondaryArchSuffix
+	devel:libgmp$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:libltdl$secondaryArchSuffix
+	devel:libncurses$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:awk
+	cmd:bison
```

Comment:
bison, flex, aclocal, autoconf are only required when building from VCS, they are *not* necessary when building from a tarball which was done here (when building from VCS you'd also need sed, grep, help2man and texinfo).
Which parts need `awk` and `find`?

Reply:
Well that's a generic information, does it apply to Haiku? Have you tried to build the recipe with your suggestions?

Reply:
I have never build any haiku recipe, I came here from a mailing list entry to the gnucobol-bug list only and wanted to give some general advice concerning GnuCOBOL as I'm... a bit experienced to build it with different systems.
As said below: they are not needed at all if you build from a tarball, if you build from VCS then more tools will be missing; if you do want to use autogen you'll definitely need aclocal, and autoconf (if those use awk then it should be their dependency), I see no reason for bison/flex, if there aren't any broken timestamps then those two should never be triggered.

Reply:
well we're experienced at building autotools software too, thanks anyway.